shell-completion/hledger-completion.bash

```diff
@@ -281,13 +281,16 @@ _hledger() {
                 else
                     hledgerFile=${COMP_WORDS[i+1]}
                 fi
+                # Pass it through compgen to unescape it
+                hledgerFile=$(compgen -W "$hledgerFile")
                 ;;
             --rules-file)
                 if [[ ${COMP_WORDS[i+1]} == '=' ]]; then
                     hledgerRulesFile=${COMP_WORDS[i+2]}
```

Comment:
I am not sure if I should avoid the code duplication in this loop. Do you like it better as it is or like this:
```sh
    for (( i=1; i < ${#COMP_WORDS[@]} - 2; i++ )); do
        case ${COMP_WORDS[i]} in
            # Use $file as a name reference
            -f|--file)
                local -n file=hledgerFile ;;
            --rules-file)
                local -n file=hledgerRulesFile ;;
            *)
                continue ;;
        esac
        if [[ ${COMP_WORDS[i+1]} == '=' ]]; then
            file=${COMP_WORDS[i+2]}
        else
            file=${COMP_WORDS[i+1]}
        fi
        # Pass it through compgen to unescape it
        file=$(compgen -W "$file")
    done
```